Public/Src/Engine/Dll/LowPrivilegeAccountUtils.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics.ContractsLight;
+using System.Security;
+using System.Security.Cryptography;
+using System.Text;
+using JetBrains.Annotations;
+
+namespace BuildXL.Engine

```

Comment:
>BuildXL.Engine [](start = 10, length = 14)

Add a folder in Public/Src/Utilities/Utilities/LowPrivilege, and add this class into that folder.